### DISCUSSION TITLE
L09G

### DISCUSSION THREAD

**Discussion by maxhoheiser** [duhow/xiaoai-patch#46] [Q&A] [answered]
Hi Guys, I am not sure about the status of L09G - Mi Smart Speaker (global). I found this discussion (https://github.com/duhow/xiaoai-patch/issues/1)

Can I use the provided boot image to build the new images?
Can I flash via ssh without modifications, or via serial soldering the 3 pin connection?

Regards Max

**Comment by duhow** [comment, marked as answer]
Hi @maxhoheiser , 

As stated in main Readme, **L09G** is currently not supported yet, since we were unable to get into a working shell or to modify any content in there.

**Comment by BrunnoMendes** [comment]
Hi, any updates on the L09G version?
I got a L09G and I'm trying to patch it with Alexa instead of Google Assistant, which is very limited and annoying to use.
Is it possible with custom firmware?

**Reply by duhow** [reply]
https://github.com/duhow/xiaoai-patch/issues/49#issuecomment-1533440769
